analytics_fetcher/ga.py

```diff
@@ -2,174 +2,6 @@
 
 """
 
-from collections import Counter
```

Comment:
Looks like this code came from https://github.com/gds-attic/search-performance-dashboard/blob/a30dcdaf9c1f707bedc82005ed580aae361aa6ee/dashboard/fetch/ga.py perhaps with the goal of doing more with the analytics. 

I don't think we need it now that position/click through data is more easily accessible in data studio (from enhanced-ecommerce).